.env.example

```diff
@@ -60,8 +60,14 @@ HEURISTAIAPIKEY     = '<add_your_heuristai_api_key_here>'
 # ANTHROPIC_API_KEY   = '<add_your_anthropic_api_key_here>'
 
 # Front end container details
-VITE_JSON_RPC_SERVER_URL  = 'http://127.0.0.1:4000/api'
-VITE_WS_SERVER_URL   = 'ws://127.0.0.1:4000'
+VITE_JSON_RPC_SERVER_URL  = 'http://127.0.0.1:4000/api' # if VITE_PROXY_ENABLED = 'true' change to '/api'
+VITE_WS_SERVER_URL   = 'ws://127.0.0.1:4000' # if VITE_PROXY_ENABLED = 'true' change to '/'
```

Comment:
@denishacquin this was the thing that wasn't working and we were hitting our heads against the wall. It was as simple as setting this to `'/'` (and also `''` worked :facepalm: )